packages/design-system-react/package.json

```diff
@@ -48,14 +48,24 @@
   },
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
+    "@storybook/react": "^8.3.5",
+    "@storybook/test": "^8.3.5",
```

Comment:
Aligns with storybook version across the monorepo 

<img width="291" alt="Screenshot 2024-10-30 at 2 31 34 PM" src="https://github.com/user-attachments/assets/af47ba35-67f6-4900-b3be-0864bb9693bd">